### PR TITLE
Add January 2021 blog post

### PR DESCRIPTION
**Pull request by dholbach**
Fixes: #66

### CODE REVIEW

**Comment by dholbach** [comment]
https://deploy-preview-70--fluxcd.netlify.app/blog/

**Comment by dholbach** [comment]
Shall we rename the blog title? What to?

Pinging @bigkevmcd and @somtochiama as well.

**Comment by dholbach** [comment]
@staceypotter too

File: content/en/blog/2021-01-06-january-update.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+0.5 release. This was due to the git library go-git used by
+source-controller not supporting specific git capabilities required by
+Azure Devops. The same requirements do not exist in the other major git
+providers, which is why this was not caught during the initial
+development of source-controller.
+
+This resulted in Flux V1 users who used Azure DevOps that were now not
+able to migrate to Flux V2. The initial attempt at a solution was to
+implement the missing capabilities in the existing library, which turned
+into its own endeavour. Instead the solution was to introduce a
```
Comment (squaremo) [inline content/en/blog/2021-01-06-january-update.md:104]:
```suggestion
into its own epic. Instead the solution was to introduce a
```

**Review by squaremo — COMMENTED** [review]
I've suggested a few corrections. For "epic" vs "endeavour", I though the former might get across the scale of the challenge a little better, but either is fine really.

File: content/en/blog/2021-01-06-january-update.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+Flagger and its copyright to fluxcd org (and thus, CNCF).
+
+Review the upcoming roadmap for Flagger - it now includes [GitOps
+Toolkit integration](https://github.com/fluxcd/flagger#roadmap).
+
+Please help us steer this project forward!
+
+Thanks also to everyone who contributed to the latest two releases:
+
+- [Flagger 1.6.0](https://github.com/fluxcd/flagger/pull/774/files):
```
Comment (stefanprodan) [inline content/en/blog/2021-01-06-january-update.md:54]: Remove the PR link please.

File: content/en/blog/2021-01-06-january-update.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+Flagger and its copyright to fluxcd org (and thus, CNCF).
+
+Review the upcoming roadmap for Flagger - it now includes [GitOps
+Toolkit integration](https://github.com/fluxcd/flagger#roadmap).
+
+Please help us steer this project forward!
+
+Thanks also to everyone who contributed to the latest two releases:
+
+- Flagger 1.6.0: Support for A/B testing using Gloo Edge HTTP headers based routing.
```
Comment (stefanprodan) [inline content/en/blog/2021-01-06-january-update.md:54]:
```suggestion
- Flagger 1.6.0:
  - Add support for A/B testing using Gloo Edge HTTP headers based routing
```

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dholbach 🍰

**Review by relu — APPROVED** [review]

**Review by phillebaba — APPROVED** [review]
